doc/encoders.texi

```diff
@@ -3564,6 +3564,22 @@ bitrate, @var{target_bitrate}, within the accuracy range @var{avbr_accuracy},
 after a @var{avbr_Convergence} period. This method does not follow HRD and the
 instant bitrate is not capped or padded.
 
+@item @var{skip_frame}
+Use per-frame metadata "qsv_skip_frame" to skip frame when encoding. This option
+defines the usage of this metadata.
+@table @samp
+@item no_skip
+Frame skipping is disabled.
+@item insert_dummy
+Encoder inserts into bitstream frame where all macroblocks are encoded as
+skipped.
+@item insert_nothing
+Similar to insert_dummy, but encoder inserts nothing into bitstream.
+@item brc_only
+skip_frame metadata indicates the number of missed frames before the current
```

Comment:
what is written into the stream ?

Reply:
According to my understanding, this affect bitrate for the rest frames. For example, if I set skip_frame for one frame, than the rest frames will be larger in size.

Reply:
What is the number of missed frames in this case ? My understanding is that skip_frame is a flag only.

Reply:
Skipframe in co2 is flag which indicate the usage of SkipFrame in enc_ctrl..
Skipframe in enc_ctrl is flag when using insert_dummy and insert_nothing mode, but in brc_only mode it is the number of missed frame.

Reply:
Got it, thanks for clarifying it.